modules/virt-checking-storage-configuration.adoc

```diff
@@ -0,0 +1,230 @@
+
+You run a storage checkup by performing the following steps:
+
+. Create a service account, role, and role bindings for the storage checkup.
+. Create a config map to provide the input to run the checkup and to store the results.
+. Create a job to run the checkup.
+. Review the results in the config map.
+. Optional: To rerun the checkup, delete the existing config map and job and then create a new config map and job.
+. When you are finished, delete the storage checkup resources.
+
+.Prerequisites
+* You have installed the OpenShift CLI (`oc`).
+* The cluster administrator has created the required `cluster-reader` permissions for the storage checkup service account and namespace, such as in the following example:
++
+[source,yaml]
+----
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRoleBinding
+  metadata:
+    name: kubevirt-storage-checkup-clustereader
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: cluster-reader
+  subjects:
+  - kind: ServiceAccount
+    name: storage-checkup-sa
+    namespace: <target_namespace>
+----
+
+
+.Procedure
+
+. Create a `ServiceAccount`, `Role`, and `RoleBinding` manifest for the storage checkup:
++
+.Example service account, role, and rolebinding manifest file
+[%collapsible]
+====
+[source,yaml]
+----
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: storage-checkup-sa
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: storage-checkup-role
+rules:
+  - apiGroups: [ "" ]
+    resources: [ "configmaps" ]
+    verbs: ["get", "update"]
+  - apiGroups: [ "kubevirt.io" ]
+    resources: [ "virtualmachines" ]
+    verbs: [ "create", "delete" ]
+  - apiGroups: [ "kubevirt.io" ]
+    resources: [ "virtualmachineinstances" ]
+    verbs: [ "get" ]
+  - apiGroups: [ "subresources.kubevirt.io" ]
+    resources: [ "virtualmachineinstances/addvolume", "virtualmachineinstances/removevolume" ]
+    verbs: [ "update" ]
+  - apiGroups: [ "kubevirt.io" ]
+    resources: [ "virtualmachineinstancemigrations" ]
+    verbs: [ "create" ]
+  - apiGroups: [ "cdi.kubevirt.io" ]
+    resources: [ "datavolumes" ]
+    verbs: [ "create", "delete" ]
+  - apiGroups: [ "" ]
+    resources: [ "persistentvolumeclaims" ]
+    verbs: [ "delete" ]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: storage-checkup-role
+subjects:
+  - kind: ServiceAccount
+    name: storage-checkup-sa
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: storage-checkup-role
+----
+====
+
+. Apply the `ServiceAccount`, `Role`, and `RoleBinding` manifest:
++
+[source,terminal]
+----
+$ oc apply -n <target_namespace> -f <storage_sa_roles_rolebinding>.yaml
+----
+
+. Create a `ConfigMap` and `Job` manifest. The config map contains the input parameters for the checkup job.
++
+.Example input config map
+[source,yaml,subs="attributes+"]
+----
+---
```

Comment:
Same comment as previous for YAML syntax.

Reply:
The syntax is correct as both the `ConfigMap` and `Job` YAML definitions are in a single file. I changed the example title to reflect that as well.